virttest/guest_agent.py

```diff
@@ -999,3 +999,11 @@ def get_virtio_device(self):
         cmd = "guest-get-devices"
         self.check_has_command(cmd)
         return self.cmd(cmd=cmd)
+
+    def get_diskstats(self):
+        """
+        Send "guest-get-diskstats", return disk stats info of guest.
```

Comment:
Please follow the correct docstring format.

Reply:
the correct command should be 'guest-get-diskstats', but CI format ask me to change it as 'disk stats' or 'disk-stats', Is that okay?

Reply:
@6-dehan That's not what I mean. Please check the above function's docstring.
```
:return: 
```

Reply:
@6-dehan A better description for the function is 'what' the function generally provides to users, instead of 'how' it does, and that's why the function named 'get_' instead of 'send_'.

Recommend description: '' Get the disk status of guest OS."